.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
rc file for lintstaged... maybe put it into package.json
at least use a consistent method to configure these tools